### PR TITLE
Fix height NGE tab

### LINKED ISSUE #9231: NGE tab is not responsive (scroll bar)
### What happened?

New interface makes the NGE tab not responsive, see scrolling bar on the right

image

Capture d'écran 2024-10-07 165258

### What did you expect to happen?

Whole window to be responsive

### How can we reproduce it (as minimally and precisely as possible)?

1. open macro tab

### On which environments the bug occurs?

Recette (SNCF)

### On which browser the bug occurs?

Firefox

### OSRD version (top right corner `Account` button > `Informations`)

staging 1482e8c

### PR DESCRIPTION
**Pull request by Uriel-Sautron**
close #9231

### CODE REVIEW

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 1]
```diff
@@ -32,21 +32,21 @@
 
     display: inline-flex;
     align-self: flex-start;
-    height: 2rem;
-    font-size: 0.875rem;
-    letter-spacing: 0.1019rem;
-    line-height: 1.125rem;
-    margin-block: 1.813rem 2.063rem;
+    height: 32px;
+    font-size: 14px;
```
Comment (clarani) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Font-size should remain in rem

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss [hunk 2]
```diff
@@ -32,21 +32,21 @@
...
+    letter-spacing: 1.6304px;
+    line-height: 18px;
+    margin-block: 29.008px 33.008px;
```
Comment (clarani) [inline front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Can you round the values ? 
```suggestion
    margin-block: 29px 33px;
```

**Review by clarani — CHANGES_REQUESTED** [review]
- can you round all the values in px ?
- can you keep rem for the font-size ?

**Comment by clarani** [comment]
Thanks for separating the 2 commits !! :star_struck:

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss (hunk 2 above)
Comment (emersion) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Right, fractional pixel values tend to cause issues and are not handled consistently by all browsers.

File: front/src/styles/scss/applications/operationalStudies/_scenario.scss (hunk 1 above)
Comment (Uriel-Sautron) [reply front/src/styles/scss/applications/operationalStudies/_scenario.scss]: Done

**Review by Akctarus — COMMENTED** [review]
Tested, works well, but to respect the choice made in the front point, all font sizes must be set to rem

**Review by clarani — CHANGES_REQUESTED** [review]
You still have some font-size in px

**Comment by Uriel-Sautron** [comment]
> You still have some font-size in px

Oupss, done thank you

**Review by clarani — APPROVED** [review]
LGTM :white_check_mark:

**Pull request merged**